extensions/amp-wordpress-embed/0.1/amp-wordpress-embed.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Embeds a Github gist
+ *
+ * Example:
+ * <code>
+ * <amp-wordpress-embed
+ *   layout="fixed-height"
+ *   data-url="https://example.com/2018/05/17/awesome-post/"
```

Comment:
This can be an actual URL like https://make.wordpress.org/core/2015/10/28/new-embeds-feature-in-wordpress-4-4/